src/components/DeleteButton.jsx

```diff
@@ -1,12 +1,12 @@
-import classNames from 'classnames';
-import './DeleteButton.scss';
+import './Button.scss';
+import Button from './Button';
 import { ReactComponent as IconDelete } from '../assets/images/ic_deleted.svg';
 
 function DeleteButton({ disable }) {
   return (
-    <button disabled={disable} className={classNames('delete-button')}>
+    <Button disable={disable} type='delete' size='36' emoji={false}>
```

Comment:
```suggestion
<Button disabled={disabled} type='delete' size='36' emoji={false}>
```
Button 컴포넌트에서 넘기고 받는 disabled 속성은 button 태그의 disabled 속성과 이름을 맞춰주는게 좋을거  같아요~!